### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'states'

### ISSUE THREAD

**Issue by alandtse** [custom-components/places#19]
It looks like there's a device update that occurs before HA is fully initialized where self.hass is none. We may have to add a check or even wait to initialize updates until after the entity is added to HA.

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/places/sensor.py", line 428, in tsc_update
    self.do_update("Track State Change")
  File "/config/custom_components/places/sensor.py", line 504, in do_update
    devicetracker_zone = self.hass.states.get(self._devicetracker_id).state
AttributeError: 'NoneType' object has no attribute 'states'
```

**Comment by bryeartem** [comment]
I am also having this issue:
```
Logger: homeassistant
Source: custom_components/places/sensor.py:504 
First occurred: September 17, 2020, 11:44:27 PM (2 occurrences) 
Last logged: September 17, 2020, 11:44:27 PM

Error doing job: Future exception was never retrieved
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/homeassistant/.homeassistant/custom_components/places/sensor.py", line 428, in tsc_update
    self.do_update("Track State Change")
  File "/home/homeassistant/.homeassistant/custom_components/places/sensor.py", line 504, in do_update
    devicetracker_zone = self.hass.states.get(self._devicetracker_id).state
AttributeError: 'NoneType' object has no attribute 'states'
```

**Comment by iantrich** [comment]
should be fixed in 1.12
